cylc/flow/ws_data_mgr.py

```diff
@@ -327,6 +382,9 @@ def generate_graph_elements(self, edges=None, graph=None,
                     graph_string=info[3],
                 )
 
+        new_points = set([get_point(p) for p in cycle_points])
```

Comment:
Does the same thing but uses less memory and takes less time:

```suggestion
        new_points = set((get_point(p) for p in cycle_points))
```

Reply:
Fixed.